specs/waku/v2/waku-filter.md

```diff
@@ -38,25 +44,22 @@ It is worth noting that a light node could get by with only using the `store`
 protocol to query for a recent time window, provided it is acceptable to do
 frequent polling.
 
-#### Protobuf
+## Protobuf
 
+<!--
 TODO Consider adding a FilterResponse acting as a form of ACK
 
-TODO Consider adding request id
-
-TODO Clarify if Messages or a list of WakuMessage are pushed
-
 TODO Specify unsubscribe mechanism and semantics
 
 TODO Investigate if we need a way to communicate (handshake?) that we are a a client - server (full node - light node) or not.
 NOTE I would imagine this is implied from the contentFilter, especially as two nodes can play multiple roles.
+-->
 
 
 ```protobuf
 message FilterRequest {
-  // space for optional request id
+  optional string topic = 1;
```

Comment:
Hmm I just realized arent we using protobuf3 here where optional is default?

Reply:
Yeah I guess we are.